Bottom: 7cc96e1654304be6bef8d35bd46de0e053801b66
Top:    ee2a8df4f5cae53c8b3835a341722597ab432d14
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 02:27:04 +0000

configure.ac: Don't copy CFLAGS into CPPFLAGS for X11 tests

CFLAGS should work in its own right, and CPPFLAGS does not need a copy.


---

diff --git a/configure.ac b/configure.ac
index 8e7a862..5b202b1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -705,7 +705,7 @@ for e in $HIDLIST; do
     case $e in
       lesstif )
 	AC_PATH_XTRA
-	CPPFLAGS="$CFLAGS $X_CFLAGS"
+	CPPFLAGS="$X_CFLAGS"
 	AC_CHECK_LIB(X11, XOpenDisplay, , , $X_LIBS)
 	AC_CHECK_LIB(ICE, main, , , $X_LIBS)
 	AC_CHECK_LIB(SM, main, , , $X_LIBS)
